tests/test_pds4_create_xml_index_blackbox.py

```diff
@@ -0,0 +1,397 @@
+from pathlib import Path
+import pytest
+import os
+import sys
+import tempfile
+sys.path.append(str(Path(__file__).resolve().parent.parent / "pds4indextools"))
```

Comment:
Doesn't pytest put the current directory into PYTHONPATH? So then you could just import this as:
```
import pds4indextools.pds4_create_xml_index as tools
```
and avoid modifying the `sys.path`.